moviesurfer/cli/main.py

```diff
@@ -1,2 +1,16 @@
-def main():
-    print("Hello from CLI, functionality is to be added")
+import click
+
+from moviesurfer.data.make_dataset import make_dataset
+
+
+@click.command()
+@click.argument("input_filepath", type=click.Path(exists=True))
+@click.argument("output_filepath", type=click.Path())
+def main(input_filepath, output_filepath):
```

Comment:
`main` is just an entry point, it is not supposed to do anything. It can print a help message, but that's all. 
For example. there is `git` command. When you run it, it just prints all the other sub-commands available. Only when you type `git add` , then you perform smth. Please, remove `click.argument`. Downloading data should be implemented as a sub-command
```bash
> moviesurfer data download
```

Reply:
Yes, you are on the right track now. I propose leaving additional complications out of the scope (like this -force one). Setting up basic interface is of higher priority right now.